requirements.txt

```diff
@@ -1,19 +1,17 @@
-# Used by the library (versions set to mirror edX)
-numpy==1.6.2
-pyparsing==2.0.7
-scipy==0.14.0
-six==1.11.0
+# Used by the library
+numpy
+scipy
+pyparsing
+six
 
 # For testing
```

Comment:
Testing and documentation can be unpinned.

Was mock moved into the standard library?

Reply:
Yes, mock comes from https://docs.python.org/3/library/unittest.mock.html.